src/opnsense/mvc/app/models/OPNsense/Routes/Route/Migrations/M0_0_1.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace OPNsense\Routes\Route\Migrations;
+use OPNsense\Base\BaseModelMigration;
+class M0_0_1 extends BaseModelMigration
+{
+    public function run($model)
+    {
+       $model.save();
```

Comment:
@AdSchellevis This is what I am thinking about how that is done. I made the model in a way it will not change any of the values so nothing breaks in mean time.